docs/src/modules/java/pages/quickstart-template.adoc

```diff
@@ -203,10 +203,13 @@ Scala::
 +
 [source,bash]
 ----
-sbt compile Test/compile
+sbt -DonlyUnitTest test
 ----
+By default the integration and unit tests are both triggered with `sbt test`. To invoke only the unit tests run `sbt -DonlyUnitTest test` or set up that value to `true` in the sbt console by `set onlyUnitTest := true`. Then run `sbt test`
```

Comment:
is this only shown when the Scala tab is selected?

Reply:
I added [.group-scala]#